gradle.properties

```diff
@@ -3,7 +3,7 @@ org.gradle.daemon=false
 
 modId=structurize
 modGroup=com.ldtteam
-modVersion=0.0.1-LOCAL
```

Comment:
No, for this purpose you can mark your dependency as changing, we don't use the `Snapshot` moniker since this has consequences for releases and other artifacts which are build locally.

Reply:
That's not a good solution either, because Minecolonies doesn't do that.  It also doesn't work without some additional code that's also not in the build scripts.  At least not when I tried it.

Releases etc should be overriding that build version on the command line anyway, so it shouldn't matter.

Reply:
What do you mean with "Minecolonies does not do that"?

Reply:
It does not mark the dependency as changing, nor does it include the extra code needed to make gradle not cache changing dependencies regardless (which is a new thing in 1.18 gradle, it appears).

Having said that, I've reverted this change for now (since it's not directly related to the rest of the PR), but I still strongly believe that it is the more correct solution and it should still be used.

Reply:
See [the gradle docs](https://docs.gradle.org/current/userguide/dynamic_versions.html#:~:text=By%20default%2C%20Gradle%20caches%20changing,resolve%20new%20snapshot%20versions%20earlier.) -- by default even "changing" deps are cached for 24 hours, which means that it will completely ignore your subsequent changes in the same day even if you republish (unless you remember to change the dep version on both sides every single time, which is a massive hassle).

Neither Minecolonies nor OPC includes the code to reduce the cache time.  And in my testing, reducing the caching time (even to a few seconds) is not completely reliable -- it will still sometimes use the cached version anyway.  Only using the snapshot tag will reliably always use the latest code.